### PR TITLE
Porting Linear Algebra Solver to V1

### PR DESCRIPTION
**Pull request by jd-bartlett96**
Conjugate gradient edsl-based solver. Should work for any positive-definite system but converges particularly quickly for finite element analysis problems.

Issues with correctness & performance persist on GPU

### CODE REVIEW

**Comment by jd-bartlett96** [comment]
@flaub @jbruestle 

I've run into what appears to be a bug in GPU kernel compilation; I'm getting incorrect results when running on the GPU for programs that run fine on the CPU. I've committed the most minimal example of this issue I could produce, which can be run with 
`bazelisk run //networks/scitile/LAS:min_ex`

File: networks/scitile/LAS/conjgrad.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+def cg_solve(np_A, np_r, timing=False, resthrsh=RESIDUAL_THRESHOLD):
+    N = len(np_r)
+
+    np_x = np.zeros(N)
+    np_rsq = np.matmul(np_r, np_r)
+    np_p = np_r.copy()
+
+    dtype = plaidml.DType.FLOAT32
+
+    A = edsl.Placeholder(dtype, np_A.shape)  #edsl.Tensor(edsl.LogicalShape(dtype, np_A.shape))
```
Comment (tzerrell) [inline networks/scitile/LAS/conjgrad.py:31]: I'm not sure I understand what this comment is telling me.

**Review by tzerrell — COMMENTED** [review]
Looks reasonable to me as a tested proof of concept. Minor inline remark.

File: networks/scitile/LAS/conjgrad.py (hunk 1 above)
Comment (jd-bartlett96) [reply networks/scitile/LAS/conjgrad.py:31]: Old comment; should be deleted.

File: networks/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+add_subdirectory(scitile/LAS)
```
Comment (tzerrell) [inline networks/CMakeLists.txt:1]: Add a newline to the end of this file. (The red minus in a circle GitHub added is indicating that no terminal newline is present.)

**Review by tzerrell — COMMENTED** [review]
LGTM, one minor inline comment.

Also this will be easier to get through CI if https://github.com/plaidml/plaidml/pull/1764 is landed and you merge that update into this PR.

**Review by tzerrell — APPROVED** [review]

**Review by tzerrell — APPROVED** [review]

**Pull request merged**